tv_apps/browser/js/awesomescreen.js

```diff
@@ -572,6 +581,49 @@ var Awesomescreen = {
     evt.target.classList.add('active');
   },
 
+  handleTopListKeydown: function awesomescreen_handleTopListKeydown(ev) {
+    if (ev.keyCode === KeyEvent.DOM_VK_RETURN) {
+      this.clickTopAction(ev);
+    }
+  },
+
+  handleTopListKeyup: function awesomescreen_handleTopListKeyup(ev) {
```

Comment:
update as following:
handleTopListKeyup: function awesomescreen_handleTopListKeyup(ev) {
    if (ev.keyCode === KeyEvent.DOM_VK_RETURN) {
        ev.button = 0;
        this.clickTopList(ev);
    }
},